model/table_metadata.py

```diff
@@ -0,0 +1,32 @@
+from model.index import *
+
+class TableMetaData:
+    def __init__(self, table_schema, table_object):
+        self.__table_schema = table_schema
+        self.table_name = table_schema[SchemaKeys.NAME]
+        self.primary_key = table_schema[SchemaKeys.PRIMARY_KEY]
+        self.columns = table_schema[SchemaKeys.COLUMNS]
+        self.__path = table_object.get_path()
+        self.indeces = self.__create_indices()
+        
+    def get_path(self):
+        return self.__path
+        
+    def serialize(self):
+        self.__serialize_table_schema()
+        self.__serialize_indices()
+
+    def __serialize_table_schema(self):
+        with open(os.path.join(self.__path, self.table_name + FILE_EXTENSION), 'w') as file:
+            json.dump(self.__table_schema, file)
```

Comment:
I suggest serializing `self.__dict__ ` instead of `self.__table_schema`
because u will use it to deserialize